corehq/apps/ota/views.py

```diff
@@ -58,9 +59,11 @@ def get_restore_response(domain, couch_user, since=None, version='1.0',
                             status=401)
 
     project = Domain.get_by_name(domain)
+    app = Application.objects.get(app_id) if app_id else None
     restore_config = RestoreConfig(
         project=project,
         user=couch_user.to_casexml_user(),
+        app=app,
         params=RestoreParams(
```

Comment:
did you consider putting `app` or `app_id` inside the `RestoreParams` instead and if so any reason you decided not to?  One of the purposes of that object is to not have to add new arguments to all the restore functions when we add new parameters which would sort of fit the use case here (though I could also see the position that `app` is "special" like `user`.